notification-portlet-api/src/main/java/org/jasig/portlet/notice/NotificationResponse.java

```diff
@@ -168,6 +169,17 @@ public NotificationResponse filterErrors(Set<Integer> hiddenErrorKeys) {
         return rslt;
     }
 
+    /**
+     * Provides the total number of notifications contained in the response.
+     */
+    @JsonIgnore
+    @XmlTransient
+    public int size() {
+        final MutableInt rslt = new MutableInt(0);
+        categories.forEach(category -> rslt.add(category.getEntries().size()));
```

Comment:
Maybe stream sum could help? Something like:

```
Integer totalNotifications = categories
    .stream()
    .mapToInt(NotificationCategory::getEntries::size)
    .sum();
```

Reply:
Yeah that's perfect.  I looked for a way to add numbers together with streams... didn't find anything.